Makefile

```
@@ -2,8 +2,12 @@
 DEV_KIT_VERSION := v1.0.7
 -include common.mk
 common.mk:
```

checkmake (0.3.2)

[warning] Target body for "common.mk" exceeds allowed length of 5 lines (6).

(maxbodylength)